packages/discovery/package.json

```diff
@@ -51,6 +51,8 @@
     "node": ">=18"
   },
   "dependencies": {
+    "@waku/interfaces": "0.0.22",
```

Comment:
nit: this is `dev` dependency always (except when variables from there are exported)

Reply:
https://github.com/waku-org/js-waku/actions/runs/8231612596/job/22507288390?pr=1876

Reply:
ok, let's keep it like this for now and figure it out later https://github.com/waku-org/js-waku/pull/1904#discussion_r1520506480